src/devices/schneider_electric.ts

```diff
@@ -565,6 +565,26 @@ const fzLocal = {
             return ret;
         },
     } satisfies Fz.Converter,
+    ias_smoke_alarm_1_heat_alarm_2_with_hush: {
```

Comment:
could you extend and use: https://github.com/Koenkk/zigbee-herdsman-converters/blob/2d229ffdae8f31c9dc1825ffa1ee4940d9cd029c/src/lib/modernExtend.ts#L1487 instead? 

Example usage: https://github.com/Koenkk/zigbee-herdsman-converters/blob/2d229ffdae8f31c9dc1825ffa1ee4940d9cd029c/src/devices/adeo.ts#L439

Reply:
As in make a local version of the iasZoneAlarm modern extend that can do the hush mode bit decode?

Reply:
Add the functionality to `modernExtend.ts`

Reply:
No worries, done! Have tested and it correctly configures the reporting and binding as before. Only slight negative is no ability to tell the user that the second alarm is heat while the first is smoke.

![image](https://github.com/user-attachments/assets/e91a5885-37d8-45d4-aa19-b8c052efc2c3)

One option would be to add a new argument maybe called secondaryZoneType?

https://github.com/Koenkk/zigbee-herdsman-converters/blob/2d229ffdae8f31c9dc1825ffa1ee4940d9cd029c/src/lib/modernExtend.ts#L1480-L1486

I think it is also acceptable as is, at least both alarms are exposed.